myning/chapters/mine/screen.py

```diff
@@ -76,9 +79,29 @@ def on_mount(self):
     def action_skip(self):
         if self.abandoning:
             self.abandoning = False
+            if isinstance(self.action, MineralAction):
+                self.action.game.toggle_paused()
+                self.update_screen()
         elif self.should_exit:
             self.exit()
-        elif isinstance(self.action, ItemAction):
+        elif isinstance(self.action, MineralAction):
+            match self.action.game.score:
+                case MiningScore.GREEN:
+                    color = "lime"
+                    trip.seconds_passed(self.action.duration)
+                case MiningScore.YELLOW:
+                    color = "yellow"
+                    trip.seconds_passed(self.action.duration)
+                case MiningScore.ORANGE:
+                    color = "orange"
+                case MiningScore.RED:
+                    color = "red"
+                    trip.seconds_passed(-10)
+                    for member in player.army.living_members:
```

Comment:
Does this break any math if the time remaining becomes higher than the original trip length? -5% complete is fine as long as we don't get any errors.

Reply:
Nope, if you start at 15 minutes and keep hitting red, your timer can increase to 15m 10s but the progress will still show 0%